### ISSUE TITLE
Map doesn't display correctly in Alien vs predator

### ISSUE THREAD

**Issue by fragmental** [djipi/Virtual-Jaguar-Rx#23]
Hello.  I apologize if this is out of place, but I can't find anywhere to report this to upstream, and it seems to be inactive.  

In Alien vs Predator, the map doesn't display correctly, which makes the game very difficult to play.

**Comment by djipi** [comment]
Hi. Sorry for this trouble; for what I know, this issue is a legacy one and not specific to the Rx version.
If a day I understand the origin of the problem, I will try to fix it.
You may also try to run the game using the Phoenix emulator or the Libreto emulator.

**Comment by fragmental** [comment]
Yes, it's in Virtual Jaguar.  I assume it's also in the Libretro/Retroarch core, but there's no way for me to tell, because Retroarch is missing the input for the map button. 

The map does work correctly in Phoenix Emulator, though.

**Comment by djipi** [comment]
For the record, it will be nice if you can add one screenshot with the problem and another screenshot/screen capture showing how it should look like (it can come from Phoenix or a real console).

**Comment by Cee123** [comment]
![Alien vs Predator (World)@Sat April 11 2020 11-10-55-970](https://user-images.githubusercontent.com/43462099/79042211-39e3fc00-7c28-11ea-91e9-dfba110d1f36.jpg)

This is how the map is supposed to display on Alien versus Predator. Screenshot taken from the Phoenix Emulator.

**Comment by djipi** [comment]
![2020-04-15 03_01_12-Virtual Jaguar v2 1 3 Rx - Now playing_ Alien vs Predator (World)](https://user-images.githubusercontent.com/1195657/79308711-18547000-7ec7-11ea-8dac-f33a15d3528b.jpg)
Screenshot taken from the Virtual Jaguar Rx version.

![2020-04-15 03_08_37-Virtual Jaguar GIT 20160613 - Now playing_ Alien vs Predator (World)](https://user-images.githubusercontent.com/1195657/79308714-19859d00-7ec7-11ea-98fa-ad243326de0c.jpg)
Screenshot taken from the Virtual Jaguar legacy version.

**Comment by fragmental** [comment]
@djipi I don't think you have the map open in those screenshots.  Either that or there wasn't any map data to display, because it's an automap and you hadn't gone anywhere yet. The map control button is Numpad 8, or whatever you have that mapped too on your controller.

There should be something there, even though it's just garbage data.  Here's an example:
![](https://i.imgur.com/NlvSc1g.png)
Here's another example
![](https://i.imgur.com/Kl9OycW.png)

The legacy version looks the same.

The garbage data sort of sticks in one third of the display, and moves and changes instead of being centered.  But none of it is recognizable as any sort of map.
